canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.jsx

```diff
@@ -575,6 +575,7 @@ class FlexibleTable extends React.Component {
 							disabled={disabled}
 							size="sm"
 							labelText={searchBarLabel}
+							aria-labelledby={`${this.props.tableLabel}-search-label`}
```

Comment:
This didn't fix the violation. [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-labelledby) is supposed to be an element in DOM.